library/build.gradle

```diff
@@ -24,6 +24,7 @@ android {
 }
 
 dependencies {
+    implementation rootProject.ext.threeTenAbp
```

Comment:
Im curious if we should leak this through our api.

Reply:
Absolutely not.

Reply:
Ok, we might have to if we remove `CalendarDay` later on however.